sample.config.yaml

```diff
@@ -90,3 +90,28 @@ logging:
       # number of bytes, or units of kb, mb, and gb. If using the units, add
       # 'k', 'm', or 'g' as the suffix
       maxSize: 50m
+
+namePatterns:
+  # The default displayname for a bridged user
+  #
+  # Available variables:
+  #
+  # name: username of the user
+  # team: name of the team
+  user: :name
+
+  # Name of bridged Slack channels
+  #
+  # Available variables:
+  #
+  # name: name of the channel
+  # team: name of the team
+  # type: type of the room
+  room: :name
```

Comment:
This isn't the default.

Reply:
I see. Can you tell me what is the default so I can update it accordingly?

Reply:
Thesea re the defaults https://github.com/Sorunome/mx-puppet-slack/blob/master/src/index.ts#L61-L64

Reply:
Thank you very much! I changed it according to the code @Sorunome linked. However I don't understand what it does. :( Would any of you be so kind the enlighten me?

Reply:
`[if?then,else]` are if-conditions in the name patterns